.travis.yml

```
@@ -1,7 +1,7 @@
 language: node_js
 sudo: required
 node_js:
-  - '12'
+  - '11'
```

Hello Ange!

Sorry, it took me a while to get back to you.
Please revert this change back to `12` and I will be more than happy to merge this PR.
👍 

```suggestion
  - '12'
```